docs/Remove-NTFSAccess.md

```diff
@@ -0,0 +1,223 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Remove-NTFSAccess
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+### PathComplex (Default)
+```
+Remove-NTFSAccess [-Path] <String[]> [-Account] <IdentityReference2[]> [-AccessRights] <FileSystemRights2>
+ [-AccessType <AccessControlType>] [-InheritanceFlags <InheritanceFlags>]
+ [-PropagationFlags <PropagationFlags>] [-PassThru] [<CommonParameters>]
+```
+
+### PathSimple
+```
+Remove-NTFSAccess [-Path] <String[]> [-Account] <IdentityReference2[]> [-AccessRights] <FileSystemRights2>
+ [-AccessType <AccessControlType>] [-AppliesTo <ApplyTo>] [-PassThru] [<CommonParameters>]
+```
+
+### SDSimple
+```
+Remove-NTFSAccess [-SecurityDescriptor] <FileSystemSecurity2[]> [-Account] <IdentityReference2[]>
+ [-AccessRights] <FileSystemRights2> [-AccessType <AccessControlType>] [-AppliesTo <ApplyTo>] [-PassThru]
+ [<CommonParameters>]
+```
+
+### SDComplex
+```
+Remove-NTFSAccess [-SecurityDescriptor] <FileSystemSecurity2[]> [-Account] <IdentityReference2[]>
+ [-AccessRights] <FileSystemRights2> [-AccessType <AccessControlType>] [-InheritanceFlags <InheritanceFlags>]
+ [-PropagationFlags <PropagationFlags>] [-PassThru] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
```

Comment:
Removes TrainerJami's FullControl ACE from the location \\itinfpw022\itxxhome\JAPE004